src/fides/api/ctl/sql_models.py

```diff
@@ -579,3 +579,15 @@ class CustomField(Base):
     )
 
     UniqueConstraint("resource_type", "resource_id", "custom_field_definition_id")
+
+
+class AuditLogResource(Base):
+    """The log of user actions against fides resources."""
+
+    __tablename__ = "audit_log_resource"
+
+    user_id = Column(String, nullable=True, index=True)
+    request_path = Column(String, nullable=True)
```

Comment:
This could give out-of-date information if we ever changed the URLs around in Fides — I wonder if we can resolve this field to some constant representing the endpoint name? Probably something to log for a future update.

Reply:
Ah, ya I can see that making sense for sure. I do still sort of like the traceability as to exactly what endpoint was hit at what point in time. If the data is still planned to be life-limited it might be moot as well

Reply:
Maybe it makes sense to at least include the field in the migration for now and then we can add it in later as needed - would that work?

Reply:
seems like a reasonable idea, but given some time sensitivity with the release, let's tackle as a follow up to get this merged 😄